### PR TITLE
Copter: Notify that the radio failsafe is disabled

### PR DESCRIPTION
**Pull request by muramura**
Disables the radio fail-safe during automatic flight.
This vehicle can be operated with a radio.
The operator does not get passively that the radio failsafe is disabled.
I learned that when the radio receiver of the radio is abnormal, it notifies the active PWM.
When I manually operate the vehicle, it notifies me that the radio failsafe is disabled.
I notify the message to the GCS, which in turn notifies the operator by voice and displays the message.

https://ardupilot.org/copter/docs/radio-failsafe.html#receiver-configuration-for-low-throttle-method

AFTER
Screenshot from 2022-01-08 06-52-31

### CODE REVIEW

File: ArduCopter/AP_Arming.cpp [hunk 1]
```diff
@@ -708,6 +708,11 @@ bool AP_Arming_Copter::arm_checks(AP_Arming::Method method)
         return false;
     }
 
+    // Throttle Failsafe check
+    if (copter.g.failsafe_throttle == FS_THR_DISABLED) {
+        GCS_SEND_TEXT(MAV_SEVERITY_NOTICE, "FS_THR_ENABLE is disabled");
```
Comment (nkruzan) [inline ArduCopter/AP_Arming.cpp:713]: maybe reword a bit
```suggestion
        GCS_SEND_TEXT(MAV_SEVERITY_NOTICE, "Throttle failsafe is disabled");
```

Comment (muramura) [reply ArduCopter/AP_Arming.cpp:713]: @nkruzan san.
Thanks for the comment.
I agreed.

**Comment by muramura** [comment]
@nkruzan san
Thanks for the comment.
I agreed.

**Review by peterbarker — COMMENTED** [review]
There are vast numbers of parameters that the user might not set up.

In this case they might not actually be using RC on their vehicle, so don't need the failsafe.

This is something where @amilcarlucas 's tool is the correct thing to do.  We can't afford to check every parameter for every mistake; too much flash.

Sorry, closing this one.
